etcdctl/ctlv3/command/printer_simple.go

```diff
@@ -285,3 +285,7 @@ func (s *simplePrinter) UserList(r v3.AuthUserListResponse) {
 		fmt.Printf("%s\n", user)
 	}
 }
+
+func (s *simplePrinter) AuthStatus(r v3.AuthStatusResponse) {
+	fmt.Println("Authentication Status:", r.Enabled)
```

Comment:
Can we also print out auth revision here?

Reply:
done